### PR TITLE
Accessibility Guide

### PR DESCRIPTION
**Pull request by reitermb**
## Summary of Changes
Merges Raft Do's and Don'ts into Accessibility Guide and migrates guide to technical documentation folder.

### CODE REVIEW

**Review by ttran-hub — APPROVED** [review]
Thanks! LGTM 🚀

File: docs/Technical-Documentation/accessibility-guide.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+## State of a11y in TDP
+The [errors audit](https://hackmd.io/79rAOVzISbOvaTNv8nSpeA) tracks all outstanding accessibility issues in the TANF Data Portal, its knowledge center, and Django Admin console. 
+
+---
+
+## What to keep in mind when testing
+The full picture of what makes for *complete* accessibility testing involves every check in Accessibility Insight's Full Assessment tool, thoughtfulness around what makes for a good experience through the lens of various assistive technologies, and real-world usability testing with people experiencing disabilities. This guide isn't going to be able to deliver all that—but it will seek to lay out some illustrative examples and frequently useful questions to pose of an experience when you're testing it for conformance. 
+
+The following checks are organized via WCAG 2.x's categories. While only a few of these checks explicitly mention screen-readers or other assistive technologies nearly all of them should be checked affirmatively regardless of how you're interacting with the site (Sight & mouse, keyboard only, screenreader, etc...).
```
Comment (ADPennington) [inline docs/Technical-Documentation/accessibility-guide.md]: What does affirmative mean in the context of a11y testing @reitermb?

**Review by lfrohlich — APPROVED** [review]
LGTM

**Comment by ADPennington** [comment]
@reitermb is this ticket ready for me to sign off on?

**Comment by reitermb** [comment]
> @reitermb is this ticket ready for me to sign off on?

Should be good to go now! I cleaned up the affirmative language and got it up to date with develop.

**Review by ADPennington — APPROVED** [review]
thanks @reitermb 👍🏾

**Pull request merged**